### DISCUSSION TITLE
Are emojis supported in StereoKit UI 🤔?

### DISCUSSION THREAD

**Discussion by jackdaus** [StereoKit/StereoKit#419] [Ideas]
It would be cool to use emojis in the UI! 😀

**Comment by maluoi** [comment]
So on Windows, StereoKit does include the MDL2 font file by default! So you can [use the symbols over here](http://modernicons.io/segoe-mdl2/cheatsheet/) right out of the box :) Or light up any other symbols using a similar font file.

However, I'm still not certain where or how proper emoji work! If those _are_ in a font file somewhere, one issue that might arise is that StereoKit doesn't currently support colors in the font atlas. So there are a few obstacles to doing emoji, but it is something I've considered! Might see it eventually :)

**Reply by jackdaus** [reply]
Thanks for the info! And thanks for pointing me to the MDL2 stuff, I'll take a look at that.
